### PR TITLE
*: replace word error in warning, info, debug logs

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Word 'error' in the warning, info or debug logs can be confusing to the user. Especially when the user filters the logs.

### What is changed and how it works?
Replace them.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported variable/fields change

### CODE REVIEW

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -948,7 +948,7 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
 				zap.Uint64("txnStartTS", c.startTS))
 			return errors.Trace(err)
 		}
-		logutil.Logger(ctx).Debug("2PC succeed with error",
+		logutil.Logger(ctx).Debug("got some exceptions, but 2pc was still successful",
```
Comment (lamxTyler) [inline store/tikv/2pc.go]: `2pc` or `2PC`?

Comment (jackysp) [inline store/tikv/2pc.go]:
```suggestion
		logutil.Logger(ctx).Debug("got some exceptions, but 2PC was still successful",
```

Comment (jackysp) [reply store/tikv/2pc.go]: 2PC is better.

**Comment by rleungx** [comment]
Does PD or TiKV also need to do this?

**Pull request merged**